force-app/main/default/layouts/Course_Registration__c-Course Registration Layout.layout-meta.xml

```diff
@@ -11,17 +11,21 @@
                 <field>Name</field>
             </layoutItems>
             <layoutItems>
-                <behavior>Edit</behavior>
+                <behavior>Required</behavior>
```

Comment:
Skal kursregistrering gjøres i Salesforce eller bare på community? For ved å sette required her, så blir det KUN required hvis man oppretter fra Salesforce. Opprettes en kursregistrering på community eller via API-et, så gjelder ikke disse reglene.

Skal de gjelde overalt, så må du lage en valideringsregel.

Reply:
Dette gjelder forøvrig for alle page layout requirements du har. Det finnes følgende måter å gjøre det på:

- Page layout
    - Kan sette read only, ingenting, required
    - Gjelder kun for brukere i Salesforce
- Field-level security
    - Gjelder for absolutt ALLE (Api, flowen til André, brukere i Salesforce)
    - Ikke anbefalt når vi er flere team og hvis andre skal bruke samme objekt
- Valideringsregel
    - Kan spesifisere bruker som det skal gjelde

Reply:
Dette feltet trenger kun å være required når man oppretter deltager i SF. Screen Flowen velger automatisk verdi for brukeren når h*n oppretter deltager via Community. Grunnen til at det ikke velges automatisk verdi for brukeren ved opprettelse i SF, er at dette forårsaket Process builder jobben til å krasje.... derfor settes det til required